rust-toolchain

```diff
@@ -1 +1 @@
-stable
\ No newline at end of file
+nightly
```

Comment:
Fixes build on aarch64 (for now). We can revert to stable once we've updated bellperson.